src/index.ts

```diff
@@ -1,9 +1,21 @@
 export type Networks = 'rinkeby' | 'mainnet';
 
+type ValueOf<T> = T[keyof T];
+
+export interface Transaction {
+  to: string;
+  value: string;
+  data?: string;
```

Comment:
@rmeissner @nicosampler should this be optional or required?

Reply:
Question is if the safe react side can handle it ;)

Reply:
If it can I am also fine with nullable

Reply:
No, it cannot. So the question is: should it handle it

Reply:
In the future probably yes, but not for this first version